src/ROUTES.js

```diff
@@ -1,5 +1,11 @@
 import lodashGet from 'lodash/get';
+import {Linking} from 'react-native';
+import Onyx from 'react-native-onyx';
 import {addTrailingForwardSlash} from './libs/Url';
+import {GetAccountValidateCode} from './libs/API';
```

Comment:
Nab but it's a bit weird for ROUTES to depend on API. Is there a better place for `openSignedInLink`?

Reply:
I don't know... tried putting it in URL but got the same shitty circular dependency.

Reply:
Open to suggestions...

Reply:
App.js

Reply:
ok moved